addons/account_payment/views/account_portal_templates.xml

```diff
@@ -32,15 +119,21 @@
     </template>
 
     <template id="portal_invoice_payment" name="Invoice Payment">
-        <div class="row">
+        <div id="o_portal_invoice_payment" class="row">
```

Comment:
i don't think we have convention for that but id beginning by o_ sound weird to me, we usually use o_ for css classes to have a distinction between bootstrap and ours

Reply:
I have seen ids starting with `o_` in multiple other places
I can still change it if u think it breaks convention

Reply:
i would not use o_ personnaly for id but it's more of a opinion ahah

Reply:
I have fixed it now in a couple of Ids